anime_downloader/commands/ezdl.py

```diff
@@ -0,0 +1,202 @@
+import logging
+import os
+
+import click
+import requests_cache
+
+from anime_downloader import session, util
+from anime_downloader.__version__ import __version__
+from anime_downloader.sites import get_anime_class, ALL_ANIME_SITES
+from anime_downloader import animeinfo
+
+logger = logging.getLogger(__name__)
+
+echo = click.echo
+sitenames = [v[1] for v in ALL_ANIME_SITES]
+
+
+# NOTE: Don't put defaults here. Add them to the dict in config
+@click.command()
+@click.argument('anime_url')
+@click.option(
+    '--episodes', '-e', 'episode_range', metavar='<int>:<int>',
+    help="Range of anime you want to download in the form <start>:<end>")
+@click.option(
+    '--url', '-u', type=bool, is_flag=True,
+    help="If flag is set, prints the stream url instead of downloading")
+@click.option(
+    '--play', 'player', metavar='PLAYER',
+    help="Streams in the specified player")
+@click.option(
+    '--skip-download', is_flag=True,
+    help="Retrieve without downloading")
+@click.option(
+    '--download-dir', metavar='PATH',
+    help="Specifiy the directory to download to")
+@click.option(
+    '--quality', '-q', type=click.Choice(['360p', '480p', '720p', '1080p']),
+    help='Specify the quality of episode. Default-720p')
+@click.option(
+    '--fallback-qualities', '-fq', cls=util.ClickListOption,
+    help='Specifiy the order of fallback qualities as a list.')
+@click.option(
+    '--force-download', '-f', is_flag=True,
+    help='Force downloads even if file exists')
+@click.option(
+    '--file-format', '-ff', default='{animeinfo_anime_title}/{animeinfo_anime_title}_{provider}_{ep_no}',
+    help='Format for how the files to be downloaded be named.',
+    metavar='FORMAT STRING'
+)
+@click.option(
+    '--provider',
+    help='The anime provider (website) for search.',
+    type=click.Choice(sitenames)
+)
+@click.option(
+    '--external-downloader', '-xd',
+    help='Use an external downloader command to download. '
+         'Use "{aria2}" to use aria2 as downloader. See github wiki.',
+    metavar='DOWNLOAD COMMAND'
+)
+@click.option(
+    '--chunk-size',
+    help='Chunk size for downloading in chunks(in MB). Use this if you '
+         'experience throttling.',
+    type=int
+)
+@click.option(
+    '--disable-ssl',
+    is_flag=True,
+    help='Disable verifying the SSL certificate, if flag is set'
+)
+@click.option(
+    '--choice', '-c',type=int,
+    help='Choice to start downloading given anime number '
+)
+@click.option(
+    '--fallback-providers', '-fp', cls=util.ClickListOption,
+    help='Specifiy the order of fallback providers as a list.')
+@click.option("--skip-fillers", is_flag=True, help="Skip downloading of fillers.")
+
+@click.pass_context
+def command(ctx, anime_url, episode_range, url, player, skip_download, quality,
```

Comment:
Some of the options can be removed for `ezdl` to make it easier?

Reply:
See line 96-100 for more context. What it does is convert the episode range to a for loop using that range correctly.